packages/server/assets/core/typedefs/modelsAndVersions.graphql

```diff
@@ -344,12 +344,19 @@ extend type Subscription {
   Subscribe to changes to a project's models. Optionally specify modelIds to track.
   """
   projectModelsUpdated(id: String!, modelIds: [String!]): ProjectModelsUpdatedMessage!
+    @hasServerRole(role: SERVER_GUEST)
```

Comment:
Noticed that we hadn't added any of these security requirements to the new FE2 subs